docs/index.md

```diff
@@ -8,7 +8,11 @@ description: |-
 
 # Terraform Provider for VMware Cloud Foundation
 
+This provider works with versions of VMware Cloud Foundation up to and including 4.5.2
```

Comment:
Perhaps 4.4 to 4.5.2?

Reply:
Changed to `This provider works with VMware Cloud Foundation 4.4.0 to 4.5.2`